### ISSUE TITLE
Android Compatibility w/ this package and react-native-firebase

### ISSUE THREAD

**Issue by hzyIO** [Pushwoosh/pushwoosh-react-native-plugin#29]
Hey guys, we are using this package along with the analytics from react-native-firebase: https://github.com/invertase/react-native-firebase

The iOS implementation seems to work fine, but we are getting a warning when we attempt to use both react-native-firebase and this package, as shown in the screenshot.

![image](https://user-images.githubusercontent.com/192363/40329552-c348e15e-5cfe-11e8-82fe-e5f53509df8a.png)

I have both this package and react-native-firebase running and was able to send a test push to my device, just wanted to know the official thoughts on this warnring and what kind of issues it could create with sending / receiving notifications?

**Comment by wfhm** [comment]
@hzyIO Hey,

Seemingly it works fine, however could you please clarify how exactly you use react-native-firebase plugin? Are you using its push notification functionality? There is a chance pushes sent from Pushwoosh could be handled twice (by Pushwoosh plugin and Firebase plugin as well).

**Comment by wfhm** [comment]
@hzyIO any updates?

**Comment by hzyIO** [comment]
We have both successfully working together (pushwoosh plugin and firebase) in our prod app. We did have an issue with opening notifications crashing on some android phones. I believe this was fixed by excluding gcm from the plugin in our build.gradle - not 100% sure though, but its working now.

`   compile (project(':pushwoosh-react-native-plugin')) {
        exclude group:'com.pushwoosh:pushwoosh-gcm'
    }`
